README.md

```diff
@@ -1,37 +1,84 @@
+You will also see any lint errors in the console.
+- ```$ npm test``` - Launches the test runner in the interactive watch mode.
+- ```$ npm run eject``` - Removes the single build dependency from your project.
 
-Runs the app in the development mode.<br>
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Production
 
-The page will reload if you make edits.<br>
-You will also see any lint errors in the console.
+The following command is intended to be launched from the project root directory.
+
+- ```$ npm run build``` - Builds the app for production to the `build` folder.
+
+## Contributing
+
+If you encounter any bug using the App, please file an [issue](https://github.com/BuildIt-Poland/buddy-app/issues).
+
+Before creating your branch please create a new issue that describe the purpose of your contribution. You will be required to link it to the pull request.
+
+### Branch convention
+
+The `master` is the most up-to-date production-ready branch. Please create new branches from `master` using the following pattern:
+
+- feature/[issue number]-[short descriptive name] - the branch will introduce a new functionality. Only 1 functionality per pull request.
+- fix/[issue number]-[short descriptive name] - the branch will introduce only fixes to existing code.
+- chore/[issue number]-[short descriptive name] - the branch will introduce enhancements that don't impact the functionality of the existing code, like code refactoring. Issue number is optional.
+- doc/[issue number]-[short descriptive name] - the branch will introduce or amend documentation only. Issue number is optional.
+
+### Pull request convention
+
+Once the development is done, please open a pull request.
+
+Each pull request must have a `title` that includes the Jira/GitHub issue ID number at the beginning, a `description` and a a `checklist`. Optionally you can include `screenshots`.
+
+Please remember to use the BUD- prefix if pointing to a Jira issue.
+
+Below is an example of a properly written pull request description.
+
+```
+# [34] Settings screen typo fix
+
+## Description
+Fixes a typo on the word "Notification" in the settings screen
+
+## Checklist
+- [X] The word "Notification" is written correctly.
+```
+
+### Other reccomendations
 
-### `npm test`
+Do not add `TODO` in the code. TODOs are indications of unfinished work, so please finish your work before submitting it. If you CAN'T finish your development for any reason, please open an issue regard it, so that work can be planned and awareness is properly raised around that. Mention of the unfinished work must be added to the pull request description aside of the new issue reference.
 
-Launches the test runner in the interactive watch mode.<br>
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+Use regular `function()` instead of arrow functions at the `top level` (applies for functional components as well)
```

Comment:
I cannot see any advantage of using this convention. When we have a functional component the common syntax is with arrow functions `export default const someComponent = ()=> <div>I am a div<div/>`